corehq/apps/app_manager/tests/test_suite_session_endpoints.py

```diff
@@ -91,6 +93,7 @@ def test_child_module_form_session_endpoint_id(self):
                     <argument id="case_id"/>
                     <stack>
                     <push>
+                        <command value="'m1'"/>
```

Comment:
@shubham1ustg5 can you double check this expected result?  This is the stack generated for a form inside a child module.  Would you expect to see command entries for the parent module, child module, and form, or just the child module and form?

Reply:
Actually I realized I can test this out by making it an end of form navigation target, and I see that it does indeed have another command for the parent module.  I think I need to work more on this to achieve parity.